docs/iris/src/whatsnew/2.0a0.rst

```diff
@@ -95,6 +95,36 @@ Incompatible Changes
 
 * The :meth:`~iris.cube.Cube.lazy_data` method no longer accepts any arguments.
 
+.. admonition:: Significant Changes in Calculated Results
+
+  Due to the replacement of biggus with dask, as described above, the results
+  of certain types of calculation may have significantly different values from
+  those obtained in earlier versions.
+  This is of a much greater order than the usual small changes in floating
+  point results : it applies especially to any data with masked points, or of
+  long integer types.
+
+  Firstly, as Dask does not natively support masked arrays, Iris is now using
```

Comment:
@pp-mo As far as I've seen we've been trying to use `iris` instead of `Iris` and `dask` instead of `Dask` ...